Formula/trec_eval.rb

```diff
@@ -0,0 +1,16 @@
+class TrecEval < Formula
+  desc "Evaluation software used in the Text Retrieval Conference"
+  homepage "https://trec.nist.gov/"
+  url "https://github.com/usnistgov/trec_eval/archive/v9.0.8.tar.gz"
+  sha256 "c3994a73103ec842e12df693749584a45814c35c36dcc15f38984bd463566ba1"
+  license :public_domain
+
+  def install
+    system "make"
+    bin.install "trec_eval"
+  end
+
+  test do
+    system "make quicktest"
```

Comment:
This is going to fail because tests aren't run in the sourcetree. In addition to that, we need a test that exercises the some of the functionality of the app ass the user would interact with it. Version checks or usage checks (foo --version or foo --help) are not sufficient, as explained in the formula cookbook.

In most cases, a good test would involve running a simple test case: run #{bin}/foo input.txt.

- Then you can check that the output is as expected (with assert_equal or assert_match on the output of shell_output)
- You can also check that an output file was created, if that is expected: assert_predicate testpath/"output.txt", :exist?

Some advice for specific cases:

- If the formula is a library, compile and run some simple code that links against it. It could be taken from upstream's documentation / source examples.
- If the formula is for a GUI program, try to find some function that runs as command-line only, like a format conversion, reading or displaying a config file, etc.
- If the software cannot function without credentials, a test could be to try to connect with invalid credentials (or without credentials) and confirm that it fails as expected.
- Same if the software requires a virtual machine, docker instance, etc. to be running.

Reply:
huh, I was sure I pushed my fix before opening the PR (because I noticed this when running `brew test`). Whatever, now I pushed the amend successfully.